src/App.vue

```diff
@@ -6,10 +6,13 @@
       <div id="nav">
         <!-- Build menu based on configured app routes. -->
         <!-- Keep router-link on one line for underlining -->
-        <span v-for="(item, index) in routes" :key="item.path">
+        <span class="nav-item" v-for="(item, index) in routes" :key="item.path">
           <span>{{ index !== 0 ? " | " : "" }}</span>
           <router-link :to="item.path">{{ item.name }}</router-link>
         </span>
+        <span class="nav-item">
+          <ThemeToggler></ThemeToggler>
```

Comment:
I prefer using the short notation.
```suggestion
          <ThemeToggler/>
```